dist/connection-manager.js

```diff
@@ -0,0 +1,231 @@
+                        }
+                    }*/
+                }
+            }
+            else {
+                let msg = data.error;
+                const queryErrorPrefix = "Error parsing SQL query:";
+                if (msg.startsWith(queryErrorPrefix)) {
+                    let query = msg.substring(queryErrorPrefix.length, msg.indexOf("ERROR")).trim();
+                    const edsEvent = {
+                        type: types_1.EDSEventType.ServerQueryError,
+                        connection: self,
+                        data: undefined,
+                        code: data.code,
+                        message: msg,
+                        query: query
+                    };
+                    let querySetsAndCallbacksOnce = self.queriesToQuerySetsAndCallbacks.get(query);
+                    if (querySetsAndCallbacksOnce) {
+                        self.handleErrorListenerForMap(querySetsAndCallbacksOnce, query, edsEvent);
+                    }
+                    self.handleGlobalListener(edsEvent);
+                }
+                else {
+                    const edsEvent = {
+                        type: types_1.EDSEventType.ServerGlobalError,
+                        connection: self,
+                        data: undefined,
+                        code: data.code,
+                        message: msg
+                    };
+                    self.handleGlobalListener(edsEvent);
+                }
+            }
+        });
+        this.connection.onClose(function (event) {
+            const edsEvent = {
+                type: types_1.EDSEventType.Close,
+                connection: self,
+                data: event
+            };
+            self.handleGlobalListener(edsEvent);
+        });
+        this.connection.onError(function (event) {
+            const edsEvent = {
+                type: types_1.EDSEventType.ClientGlobalError,
+                connection: self,
+                data: event,
+                message: "Client error",
+            };
+            self.handleGlobalListener(edsEvent);
+        });
+    }
+    /**
+     * Send data directly to web socket
+     */
+    send(msg) {
+        this.connection.send(msg);
+    }
+    querySet() {
+        return new query_set_1.QuerySet(this, this.filtersState);
+    }
+    /**
+     * Disconnect from web socket
+     */
+    disconnect() {
+        this.connection.disconnect();
+    }
+    /**
+     * Get configuration of the connection
+     */
+    getConfig() {
+        return this.config;
+    }
+    /**
+     * Get connection id
+     */
+    getId() {
+        return this.connection.getId();
+    }
+    /**
+     * Check weather it connected
+     */
+    status() {
+        if (this.connection) {
+            return this.connection.status();
+        }
+        else {
+            return types_1.ConnectionStatus.Closed;
+        }
+    }
+    handleGlobalListener(edsEvent) {
+        var _a;
```

Comment:
Use let whenever possible in JS
```suggestion
        let _a;
```

Reply:
This file is not using at all, deleted